ffs/types.go

```diff
@@ -252,6 +252,8 @@ type FilConfig struct {
 	CountryCodes []string
 	// FilRenew indicates deal-renewal configuration.
 	Renew FilRenew
+	// Addr is the wallet address used to store the data in filecoin
+	Addr string
```

Comment:
This is really the root of most of the changes. Added `Addr` string to `FilConfig`